docs/releases.rst

```diff
@@ -0,0 +1,109 @@
+.. _releases:
+
+Releases
+=========
+
+There are three types of Renku releases: Sprint release, Hot/bug fix release and Quick release.
+
+Sprint release
+---------------
+
+A new release is planned on the Tuesday after the end of every sprint at 12:00
+(every three weeks).
+The current state of the main branch is released as is, only checking the
+wording in the CHANGELOG to ensure it is of good quality and adding the version
+information.
+
+Pre-requisites [Graph, UI, RP]
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Components release whenever they want, the schedule is up to them. They are
+responsible for updating the changelog in the auto-PR in the renku repository,
+in a section above the one for the most recent release.
+The auto-PR is only merged with an updated changelog/release notes.
+Ideally, components would release when there are meaningful changes or so
+there's a release at least every 3 weeks.
+
+If two components need to bee coordinated for a release (e.g. new version
+of Graph needs new version of RP), they should combine their auto-PRs and
+merge them in a single go. Generally, components should strive to keep
+dependencies between them to a minimum and coordinated component releases
+should not usually be needed(by using APi versioning, feature flags etc.).
+
+If a merged auto-PR that has not yet been included in a Renku release turns
+out to be faulty, it has to be reverted in the renku repository so it's not
+accidentally included in a release. A new auto-PR with the fix can then be
+created. The main release DOES NOT wait for a fix.
+
+If a regression made it into a release, the fix must add an acceptance test
+that would catch it, within reason.
+
+Process [YAT]
+~~~~~~~~~~~~~
+
+* Make a Pull Request in renku repository with updates relevant -and adapted- to USERS and ADMINS in the CHANGELOG.md file. Ask ORDES for review when necessary.
+
+* Rollout updates
+
+    * For non-breaking changes: create two pull requests in terraform-renku repository with the changes, one for limited and one for renkulab. Important: add the scheduled maintenance label to help avoiding it being merged before time.

```

Comment:
There is one PR per deployment - but we should say here that limited can be used as a "canary" and others follow ~1 week later